prac_05/state_names.py

```diff
@@ -0,0 +1,19 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
```

Comment:
I like your formatting and having the list of all states before the user adds in their input.